src/olympia/devhub/views.py

```diff
@@ -1389,12 +1391,17 @@ def version_bounce(request, addon_id, addon, version):
 @dev_required
 def version_stats(request, addon_id, addon):
     qs = Version.objects.filter(addon=addon)
-    reviews = (qs.annotate(reviews=Count('reviews'))
-               .values('id', 'version', 'reviews'))
+    reviews = (qs.annotate(review_count=Count('reviews'))
+               .values('id', 'version', 'review_count'))
     d = dict((v['id'], v) for v in reviews)
-    files = qs.annotate(files=Count('files')).values_list('id', 'files')
-    for id, files in files:
-        d[id]['files'] = files
+    files = (
+        qs
+        .annotate(file_count=Count('files'))
+        .values_list('id', 'file_count'))
+    for id, file_count in files:
```

Comment:
good call, will do. Although I'm a strong believer that this complicates code but too many people aren't on my side of this anyway :laughing: